packages/wallet-sdk/src/relay/WalletLinkRelay.ts

```diff
@@ -197,10 +170,14 @@ export class WalletLinkRelay
       // reason we don't get a response via an explicit web3 message
       // we can still fulfill the eip1102 request.
       Array.from(WalletLinkRelay.accountRequestCallbackIds.values()).forEach((id) => {
-        const message = Web3ResponseMessage({
+        const message: WalletLinkEventData = {
+          type: 'WEB3_RESPONSE',
           id,
-          response: RequestEthereumAccountsResponse([selectedAddress as AddressString]),
-        });
```

Comment:
no more `RequestEthereumAccountsResponse`
it's just  `Web3Response` with `method: requestEthereumAccounts`